benchmarks/benchmark_suite/scripts/run_benchmark.bash

```diff
@@ -1,14 +1,33 @@
 #!/bin/bash
-SCRIPT_PATH=$1
-KOKKOS_DEVICES=$2
-KOKKOS_ARCH=$3
-COMPILER=$4
-if [[ $# < 4 ]]; then
-  echo "Usage: ./run_benchmark.bash PATH_TO_SCRIPTS KOKKOS_DEVICES KOKKOS_ARCH COMPILER"
-else
+readonly SCRIPT_PATH=$1
+readonly KOKKOS_DEVICES=$2
+readonly KOKKOS_ARCH=$3
+readonly COMPILER=$4
+if [[ $# -lt 4 ]]; then
+  echo "Usage: ./run_benchmark.bash PATH_TO_SCRIPTS KOKKOS_DEVICES KOKKOS_ARCH COMPILER"; exit 1
+fi
+
+function get_logfile() {
+  local logfile=run-benchmark--$(date +"%Y-%m-%d-%H-%M").log
+
+  cd ${PWD}/kokkos
+  local git_info="KOKKOS_BRANCH: $(git branch --show-current)
```

Comment:
`git branch --show-current` requires Git 2.22. I can use `rev-parse` here if that's too fresh for some reason.